_search-plugins/knn/knn-vector-quantization.md

```diff
@@ -157,3 +163,45 @@ As an example, assume that you have 1 million vectors with a dimension of 256 an
 1.1 * (((2 * 256) * 1,000,000) + (4 * 128 * 256))  ~= 0.525 GB
 ```
 
+## Faiss product quantization
+
+Product quantization is a technique that allows users to represent a vector in a configurable amount of bits. In general, it can be used to achieve a higher level of compression compared to byte and scalar quantization. Product quantization works by breaking up vectors into _m_ subvectors, and  encoding each subvector with _code_size_ bits. Thus, the total amount of memory for the vector ends up being `m*code_size` bits, plus overhead. For more details about the parameters of product quantization, see [PQ parameters]({{site.url}}{{site.baseurl}}/search-plugins/knn/knn-index/#pq-parameters). Product quantization is only supported for the _Faiss_ engine and can be used with either the _HNSW_ or the _IVF_ ANN algorithms.
```

Comment:
```suggestion
PQ is a technique used to represent a vector in a configurable amount of bits. In general, it can be used to achieve a higher level of compression compared to byte and scalar quantization. PQ works by breaking up vectors into _m_ subvectors and  encoding each subvector with _code_size_ bits. Thus, the total amount of memory for the vector ends up being `m*code_size` bits, plus overhead. For details about the parameters, go to [PQ parameters]({{site.url}}{{site.baseurl}}/search-plugins/knn/knn-index/#pq-parameters). PQ is only supported for the _Faiss_ engine and can be used with either the _HNSW_ or the _IVF_ ANN (Artificial Neural Network) algorithms.
```

Reply:
ANN stands for Approxximate Nearest Neighbor. Ill fix